extensions/target-specific/chromium/blink/blink-helpers/blink-helpers.js

```diff
@@ -480,21 +480,33 @@ Loader.OnLoad(function() {
         .thenAll((first, second) => `{${layoutValueToPx(first)}, ${layoutValueToPx(second)}}`);
     }));
 
+    // Old name for PhysicalSize
     DbgObject.AddTypeDescription(Chromium.RendererProcessType("blink::NGPhysicalSize"), "Size", true, UserEditableFunctions.Create((size) => {
         return Promise.all([size.f("width").val(), size.f("height").val()])
         .thenAll((first, second) => `{${layoutValueToPx(first)}, ${layoutValueToPx(second)}}`);
     }));
 
+    DbgObject.AddTypeDescription(Chromium.RendererProcessType("blink::PhysicalSize"), "Size", true, UserEditableFunctions.Create((size) => {

```

Comment:
I think we can avoid writing the extended field twice by using a function that regex matches both blink::NGPhysicalSize and blink::PhysicalSize (like we do in stl-helpers).

Reply:
Done. I thought Chromium.RendererProcessType would be a problem but it wasn't.